keyboards/crkbd/keymaps/mvi/config.h

```diff
@@ -0,0 +1,129 @@
+/*
+This is the c configuration file for the keymap
+
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+Copyright 2015 Jack Humbert
+Copyright 2021 Emmanuel Vill�ger
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* Select hand configuration */
+
+#define MASTER_LEFT
+// #define MASTER_RIGHT
+// #define EE_HANDS
+
+// #define SSD1306OLED
+#undef USE_I2C
+#undef SSD1306OLED
+
+#define USE_SERIAL_PD2
+
+#ifdef OLED_DRIVER_ENABLE
+   // Time unused before switching off oled screeen
+   #define OLED_SWITCHOFF 30000
```

Comment:
Per the docs, this should be: 
https://docs.qmk.fm/#/feature_oled_driver?id=basic-configuration

```suggestion
   #define OLED_TIMEOUT 30000
```